control/init.sh

```diff
@@ -39,6 +39,7 @@ alias update_code='~/commcarehq-ansible/control/update_code.sh && . ~/init-ansib
 
 [ ! -f ~/.bash_completion ] && source  ~/commcarehq-ansible/control/.bash_completion
 cp ~/commcarehq-ansible/control/.bash_completion ~/
```

Comment:
Unrelated to this PR: this will overwrite `~/.bash_completion` if it exists. I'm not sure that's a good idea since it's not a particularly project-specific name and people tend to not always have backups of all their files...

Reply:
Yeah, seems better to just replace these two lines with `source  ~/commcarehq-ansible/control/.bash_completion`, right?

Reply:
yes